tests/components/kodi/__init__.py

```diff
@@ -0,0 +1,28 @@
+"""Tests for the Kodi integration."""
+from homeassistant.components.kodi.const import DOMAIN
+from homeassistant.const import CONF_HOST
+
+from .util import MockConnection
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+
+
+async def init_integration(hass) -> MockConfigEntry:
+    """Set up the Kodi integration in Home Assistant."""
+    entry_data = {
+        CONF_HOST: "1.1.1.1",
+    }
+    entry = MockConfigEntry(domain=DOMAIN, data=entry_data)
+    with patch(
+        "homeassistant.components.kodi.Kodi.ping",
+        return_value=True,
+    ), patch(
```

Comment:
```suggestion
    ), patch(
        "homeassistant.components.kodi.Kodi.get_application_properties",
        return_value={"version": {"major": 1, "minor": 1}},
    ), patch(
```

Reply:
@ctalkington I fixed the test

Reply:
oh, but now we have a conflict :)

Reply:
sorry, i hope im mpre help than a pain. just trying to help get a number of PRs through with a little downtime i had this weekend :)

Reply:
really appreciated. Feel free to guide me though, instead of having to code through Github :)